curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9ebb7b12bca3025b0a935.md

```diff
@@ -21,19 +21,19 @@ const clearForm = code.split("function clearForm()")[1];
 assert.match(clearForm, /for\s*\(/);
```

Comment:
i think we should remove it because when the user only writes `for(){}` our editor won't pick it up due to syntax error and they won't even get hints. 
```suggestion

```